rfc/text/advanced/ap_38_transport_rawsocket.md

```diff
@@ -221,16 +222,32 @@ The serialized octets for a message to be sent are prefixed with exactly 4 octet
 {align="left"}
         MSB                                 LSB
         31                                    0
-        RRRR RTTT LLLL LLLL LLLL LLLL LLLL LLLL
+        RRRR XTTT LLLL LLLL LLLL LLLL LLLL LLLL
 
-The *first octet* has the following structure
+The *three least significant octets* constitute an unsigned 24 bit integer that provides the length of transport message payload following, excluding the 4 octets that constitute the prefix.
```

Comment:
I start with the description of the L bits here so that the description of the X bit below is easier to understand.